storage/indexer/src/db_v2.rs

```diff
@@ -250,11 +249,11 @@ impl<'a, R: StateView> TableInfoParser<'a, R> {
     }
 
     fn parse_struct(&mut self, struct_tag: StructTag, bytes: &Bytes) -> Result<()> {
```

Comment:
can we rename these methods to `process_table_infos_in_struct`, `process_table_infos_in_resource_group`, etc.? otherwise during review one has to look at code around to understand why `parse_struct` collects table info